package/spec/frontend/media/media_spec.js

```diff
@@ -0,0 +1,75 @@
+import {media, VideoPlayer} from 'pageflow/frontend';
+import '$support/fakeBrowserFeatures';
+
+describe('media', function() {
+  let fileSource = {
+    id: 102,
+    permaId: 5,
+    width: 1000,
+    height: 500,
+    urls: {
+      ogg: 'http://example.com/example.ogg',
+      m4a: 'http://example.com/example.m4a',
+      mp3: 'http://example.com/example.ogg'
+    },
+  }
+  describe('#getPlayer', function() {
+    let player = media.getPlayer(fileSource, {
+      type: 'audio',
+      playsInline: true,
+      loop: true
+    });
```

Comment:
Never put test code directly into `describe` blocks. If `getPlayer` causes an exception, the whole test suite will crash. If you want to share test code between tests, you can use a `beforeEach` block. As a default, I'd rather duplicate code like this in all test cases to make them more self contained. 

Tests in the Pageflow repository normally follow the so called triple A structure: Arrange, Act, Assert.

These three blocks should ideally be easy to spot when reading a text, for example by separating them with empty lines.

The Arrange block, sets everything up for the test. It creates dependencies and fixture data that are required to run the code we actually want to test. If there is repetitive setup code, we can consider extracting it to a `beforeEach` block, but should - if possible - create a test helper instead to prevent introducing [mystery guests](https://thoughtbot.com/blog/mystery-guest).

The act block runs the code we want to test. It should never be extracted to a `beforeEach` block since it's the core of the test.  Also extracting the act part often leads to mixing up test cases. Even in this simple example, the `media.getPlayer` call (which is the act part of these tests) contains details related to two different tests: the `loop` option and the `playsinline` option. Also extracting the act block mostly does not scale. What if want to add a test case for when the `loop` option is missing? We will no longer be able to use a common act block.

The assert block, contains the expectations about what the act block did.